src/main/groovy/com/matthewprenger/cursegradle/CurseRelation.groovy

```diff
@@ -6,25 +6,40 @@ import static com.matthewprenger.cursegradle.Util.check
 
 class CurseRelation implements Serializable {
 
-    private void addRelation(String typeIn, String slugIn){
+    private void addRelation(String typeIn, String slugIn) {
         projects.add(new Project(type: typeIn, slug: slugIn))
     }
 
+    @Deprecated
     void requiredLibrary(String slugIn) {
-        addRelation("requiredLibrary", slugIn)
+        addRelation("requiredDependency", slugIn)
     }
+    void requiredDependency(String slugIn) {
+        addRelation("requiredDependency", slugIn)
+    }
+    @Deprecated
     void embeddedLibrary(String slugIn) {
-        addRelation("embeddedLibrary", slugIn)
+        addRelation("embeddedDependency", slugIn)
+    }
+    void embeddedDependency(String slugIn) {
+        addRelation("embeddedDependency", slugIn)
     }
+    @Deprecated
     void optionalLibrary(String slugIn) {
-        addRelation("optionalLibrary", slugIn)
+        addRelation("optionalDependency", slugIn)
+    }
+    void optionalDependency(String slugIn) {
+        addRelation("optionalDependency", slugIn)
     }
     void tool(String slugIn) {
         addRelation("tool", slugIn)
     }
     void incompatible(String slugIn) {
         addRelation("incompatible", slugIn)
     }
+    void include(String slugIn) {
```

Comment:
While we do support include on the site I believe the API will reject this one

Reply:
@ZeldoKavira Can I get a full confirmation on whether the API will actually throw up on that one or not please? if it does, I'll comment it out till the API does actually support it.

Also, in Related, on site and therefore I assume also the API, Minecraft 1.13.1, Java 10-12 arn't available choices, while Java 11 and 12 are still EA, 11 be stable soon.

Reply:
I just ran a test build using the include relation and the API refused it with the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':curseforge223797'.
> [CurseForge 223797] Error Code 1002: Error in field `metadata`:
  * Value "include" is not defined in enum. Path 'relations.projects[0].type', line 1, position 287.
```